mock/py/mockbuild/config.py

```diff
@@ -313,6 +313,8 @@ def setup_default_config_opts(unprivUid, version, pkgpythondir):
     config_opts['stderr_line_prefix'] = ""
     config_opts['additional_packages'] = None
 
+    config_opts["no-config"] = {}
```

Comment:
Documenting this in site-defaults will be nice.

Reply:
I think that it would eventually confuse users;  this is never to be configured by users (or at least very unlikely).